pyprover-source/logic.coco

```diff
@@ -113,50 +113,22 @@ class Expr:
 
 class Top(Expr):
     """True"""
-    __slots__ = ()
+    __slots__ = ('elems')
     def __eq__(self, other) = isinstance(other, Top)
     def __repr__(self) = "top"
     def __str__(self) = top_sym
     def __bool__(self) = True
-    def __hash__(self) -> int:
```

Comment:
It looks like all of this stuff was removed, but not actually added back anywhere.

Reply:
only the stuff that i added for hash and all is removed